src/stryker-net/quickstart.pug

```diff
@@ -40,25 +40,33 @@ block content
                 ```js
                 {
                     "stryker-config":
-                    {
-                        "reporter":"Console",
-                        "logConsole":"info",
-                        "timeoutMS":10000,
-                        "logFile":true,
-                        "projectName":"ExampleProject.csproj"
+                    {      
+                        "reporter":"Console",					
+                        "log-level":"info",
+                        "timeout-ms":10000,
+                        "log-file":true,
+                        "project-file":"ExampleProject.csproj",
+                        "max-concurrent-test-runners":4,
+                        "max-concurrent-test-runners": {
+                            "threshold-options": {
```

Comment:
The treshold options have changed. They are no longer in a sub section:
```
        "threshold-high": 80,
        "threshold-low": 70,
        "threshold-break": 60,
```
Also we have more options now that displayed here. I'm thinking if we could somehow include them from the github documentation. It's not handy to keep the options up to date in two places since they are changing pretty fast.

For now I think it's best to copy them from [here](https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.CLI/README.md#use-a-config-file)